### ISSUE TITLE
Can't select right input field (regression)

### ISSUE THREAD

**Issue by elleuca** [getgauge/taiko#530]
### Expected behavior
taiko should find in page the specified input field

### Actual behavior
taiko always select and interact with the first input field in page. 

It seems a regression of master over 0.6.0 release

### Steps to reproduce

```javascript
await openBrowser();
await goto('github.com');
await highlight(inputField({id: 'user[email]'}));
await highlight(inputField({id: 'user[login]'}));
await screenshot();
```

### Screenshot with taiko 0.6.0

![Screenshot-1557263365667](https://user-images.githubusercontent.com/1573759/57334091-0f87e580-711f-11e9-88f6-2d21be229cc0.png)

### Screenshot with current master

![Screenshot-1557263596578](https://user-images.githubusercontent.com/1573759/57334165-4231de00-711f-11e9-8cd6-bbcdf3c61c45.png)

### Versions
```
node: 10.15.3

taiko (working): 0.6.0 (Chromium:73.0.3665.0) 

taiko (broken): 0.7.0 (Chromium:74.0.3723.0) 
ref: github:getgauge/taiko#9691d04a220c82b21e2d161e5d82ebf6817de3da
```

**Comment by Debashis9012** [comment]
Definitely its an issue. I am able to reproduce this with the latest master 
```
commit:  f7623da1112c4a7f5ee8b40e13d1955b0fde2d7f 
Version: 0.7.0 (Chromium:74.0.3723.0)
```
@elleuca we are looking into it.

**Comment by shubhamsc** [comment]
Fixed available in 06135731a9c1175bed934d089ac477b698c18e94

**Comment by Debashis9012** [comment]
After the fix input field selects correctly in the webpage.
Tested version: ```0613573 ```
